hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/AzureBlobFileSystemStore.java

```diff
@@ -1214,11 +1217,11 @@ private void initializeClient(URI uri, String fileSystemName, String accountName
     if (tokenProvider != null) {
       this.client = new AbfsClient(baseUrl, creds, abfsConfiguration,
           new ExponentialRetryPolicy(abfsConfiguration.getMaxIoRetries()),
-          tokenProvider, abfsPerfTracker);
+          tokenProvider, abfsPerfTracker, instrumentation);
     } else {
       this.client = new AbfsClient(baseUrl, creds, abfsConfiguration,
           new ExponentialRetryPolicy(abfsConfiguration.getMaxIoRetries()),
-          sasTokenProvider, abfsPerfTracker);
+          sasTokenProvider, abfsPerfTracker, instrumentation);
```

Comment:
At this point the variable name is instrumentation and then it AbfsClient() it is statistics. I know these classes has been created earlier but I feel the names are bit confusing. 
A few suggestions.
- AbfsCounters could be AbfsInstrumentation and AbfsInstrumentation could be AbfsInstrumentationImpl
- Let AbfsCounters be same. change AbfsInstrumentation to AbfsCountersImpl. 
It is not important to change but if everybody feels the same then sooner the better. 
CC @steveloughran